### ISSUE TITLE
Installing alongside kafka-node breaks gRPC

### ISSUE THREAD

**Issue by jwulf** [creditsenseau/zeebe-client-node-js#124]
When you install zeebe-node and kafka-node in the same project - even if you don't import kafka-node - the gRPC client in zeebe-node breaks. You see an error message in the zeebe-node log output:

```
Cannot read property 'on' of undefined
```

See the reproducer here: https://github.com/need4eat/kafka-zeebe-test

It looks like kafka-node installs a version of Long that overwrites the version needed by zeebe-node's gRPC dependency.

**Comment by jwulf** [comment]
@need4eat this seems to be fixed with 0.22.1 of zeebe-node and 5.0.0 of kafka-node. Can you test again?

**Comment by need4eat** [comment]
Checked again with 0.22.1 of zeebe-node, issues still seems to persist.

**Comment by jwulf** [comment]
It depends on installation order, and it is not clear to me the order. 

Note: A working installation does _not_ produce a `package-lock.json` that makes subsequent installations work. `npm shrinkwrap` doesn't do it either... :-(

## Case 0. Installation from package.json

Fails, until zeebe-node is manually installed at the end.

```
npm i
// fails
npm i kafka-node
// fails
npm i zeebe-node
// now it works
```

This looks like installing zeebe-node _after_ kafka-node. However, I cannot replicate an install order that produces this state by manually installing the packages.

## Case 1. 

The intermediate step of installing `kafka-node` is not needed: 

```
npm i
// fails
npm i zeebe-node
// works
```

Again, this looks like installing zeebe-node _after_ kafka-node.


## Case 2. Installing kafka-node first fails

```
npm i kafka-node
npm i zeebe-node
// fails
```

Once this happens, there is no further combination of installing the packages that will cause them to work.

## Case 3. Installing zeebe-node first works

```
npm i zeebe-node
npm i kafka-node
// works
```

**Comment by jwulf** [comment]
I've npm shrinkwrapped it for alpha-2. If that doesn't address it, we'll try https://github.com/rogeriochaves/npm-force-resolutions next.

**Comment by jwulf** [comment]
I've added to an issue in kafka-node about it. It is an unmaintained package - https://github.com/cainus/BufferMaker - that they are using that locks `long`:

```
"dependencies": {
    "long": "1.1.2"
  },
```

**Comment by jwulf** [comment]
Adding long to our package deps to hoist this.

**Comment by jwulf** [comment]
This is addressed in zeebe-node 0.23.0-alpha.2. 

`long` has been added to the deps of zeebe-node, and you can now install kafka-node and zeebe-node in any order.
